crates/ruma-common/src/events/key/verification/start.rs

```diff
@@ -151,7 +151,9 @@ pub struct SasV1Content {
 
     /// The message authentication codes that the sending device understands.
     ///
-    /// Must include at least `hkdf-hmac-sha256`.
+    /// Must include at least `hkdf-hmac-sha256.v2`. Should also include `hkdf-hmac-sha256` for
+    /// compatibility with older clients, though this identifier is deprecated and will be
```

Comment:
```suggestion
    /// compatibility with older clients, though this MAC is deprecated and will be
```
? "Identifier is deprecated" sound a bit like the same algorithm is still available in non deprecated form under a different name.